lib/taskana-core/src/main/java/pro/taskana/spi/history/api/TaskanaHistory.java

```diff
@@ -20,21 +20,21 @@
   void initialize(TaskanaEngine taskanaEngine);
 
   /**
-   * Create a new task history event.
+   * Create a new {@link TaskHistoryEvent}.
```

Comment:
link or linkplain?

Reply:
I will correct it to @linkplain (according to guidelines).